src/main/jbake/content/hawkular-metrics/docs/user-guide/index.adoc

```diff
@@ -1679,6 +1679,8 @@ then needs the prefix to uniquely identify the correct metric.
 
 |===
 
+Metrics has some special support for stream-based alerting, see <<Metrics Group Triggers>> for the details.
```

Comment:
It is a matter of taste, my first reading expected to find here the info.
No blocker from my side.
Just normally jumpings make more difficult to read it but as this are is mostly to reference them I think it is fine.